composer.json

```diff
@@ -43,7 +43,7 @@
     "suggest": {
         "sonata-project/doctrine-orm-admin-bundle": "~2.3",
         "liip/imagine-bundle": "~0.9",
-        "amazonwebservices/aws-sdk-for-php": "~1.5",
+        "aws/aws-sdk-php": "~2.7",
```

Comment:
I missed the fact that the old sdk was also in the require-dev block, changed it there